### ISSUE TITLE
Failing Backward() Test for Sequential Layer.

### ISSUE THREAD

**Issue by spkanung** [spkanung/neural#1]
The layers are instantiated in the following order in layers/sequential.py:
1. Full Layer
2. ReLu Layer
3. Full Layer
4. SoftMax Layer
5. Cross Entropy Loss Layer

Forward() function simply calls the forward functions of each of the layers sequentially. The output from one layer goes into the input of the next layer.
Similarly Backward() function simply calls the backward functions of each layer sequentially, in reverse order.
